frontend/src/components/atoms/inputs/CheckboxInput.js

```diff
@@ -5,7 +5,7 @@ import Label from './Label';
 export const CheckboxInput = ({
   label,
   helper,
-  meta,
+  meta = null,
```

Comment:
Pourquoi ?

Reply:
I was using this component in a ts file, my IDE warned me that there was no default value for the meta props. As it is an optionnal value, I just put a default value to make this more clear in the props definition.

Reply:
I removed it since it's redoudant with the new type definition.